pkgs/by-name/li/libvlcpp/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  autoreconfHook,
+  pkg-config,
+  libvlc,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "libvlcpp";
+  version = "0.1.0-unstable-2024-02-04";
+
+  src = fetchFromGitLab {
+    owner = "videolan";
+    repo = "libvlcpp";
+    domain = "code.videolan.org";
+    rev = "44c1f48e56a66c3f418175af1e1ef3fd1ab1b118";
+    sha256 = "sha256-nnS4DMz/2VciCrhOBGRb1+kDbxj+ZOnEtQmzs/TJ870=";
+  };
+
+  propagatedBuildInputs = [
```

Comment:
Do we really need to propagate libvlc?

Reply:
i would say yes without it it's relatively useless